packages/create-plugin/templates/backend/pkg/plugin/datasource.go

```diff
@@ -44,7 +44,9 @@ func (d *Datasource) Dispose() {
 // The QueryDataResponse contains a map of RefID to the response for each query, and each response
 // contains Frames ([]*Frame).
 func (d *Datasource) QueryData(ctx context.Context, req *backend.QueryDataRequest) (*backend.QueryDataResponse, error) {
-	log.DefaultLogger.Info("QueryData called", "request", req)
+	// when logging at a non-Debug level, make sure you don't include sensitive information in the message
+	// (like the *backend.QueryDataRequest)
+	log.DefaultLogger.Debug("QueryData called", "request", req)
```

Comment:
In this case I'd remove the `*backend.QueryDataRequest` param completely just in case people don't think to read the comment, don't understand it or just blindly delete it 🙈

Reply:
Like this?
```suggestion
	log.DefaultLogger.Debug("QueryData called")
```

Reply:
Yeah I think that should be fine! You could also add something like:
```go
log.DefaultLogger.Debug("QueryData called", "numQueries", len(req.Queries))
```

to demonstrate how to pass key values to the log. Totally up to you 😄

Reply:
Good point! I think demonstrating how to pass key/value to the log is valuable :+1: :+1: